src/js/entities-service/entities/program-actions.js

```diff
@@ -25,19 +25,20 @@ const _Model = BaseModel.extend({
     const currentUser = Radio.request('bootstrap', 'currentUser');
     const currentOrg = Radio.request('bootstrap', 'currentOrg');
     const states = currentOrg.getStates();
-    const action = this.pick('name', 'details', '_owner', '_program', '_form');
+    const action = this.pick('name', 'details', '_owner', '_form');
     const dueDay = this.get('days_until_due');
     const dueDate = (dueDay === null) ? null : moment().add(dueDay, 'days').format('YYYY-MM-DD');
 
     _.extend(action, {
-      _patient: patientId,
-      _state: states.at(0).id,
-      _owner: action._owner || {
+      '_patient': patientId,
+      '_state': states.at(0).id,
+      '_owner': action._owner || {
         id: currentUser.id,
         type: 'clinicians',
       },
-      duration: 0,
-      due_date: dueDate,
+      '_program_action': this.id,
```

Comment:
no need to introduce the `'`s only if it needed to be `_program-action`  `_`s are ok

Reply:
ah yeah. leftover from when I was Doing It Wrong